src/parser/smt2/smt2.h

```diff
@@ -197,6 +197,24 @@ class Smt2 : public Parser
 
   void resetAssertions();
 
+  /**
+   * Creates a command that asserts a rule.
+   *
+   * @param kind The kind of rule (RR_REWRITE, RR_REDUCTION, RR_DEDUCTION)
+   * @param bvl Bound variable list
+   * @param triggers List of triggers
+   * @param guards List of guards
+   * @param heads List of heads
+   * @param body The body of the rule
+   * @return The command that asserts the rewrite rule
+   */
+  std::unique_ptr<Command> assertRule(Kind kind,
```

Comment:
Can you change this method's name to `assertRewriteRule`?

Reply:
Hahaha, I was actually going back and forth on that. Done :)